10-validation.md

```diff
@@ -1,79 +1,78 @@
+   > :eyes: After executing the command below, you should immediately return to your previous terminal and observe that the web application is responding with `HTTP 200` even during the outages.
 
-### Steps
-
-1. Browse to the site with the following appended to the URL: `?sql=DELETE%20FROM` (e.g. <$FRONTDOOR_FQDN/?sql=DELETE%20FROM>).
-1. Observe that your request was blocked by Application Gateway's WAF rules and your workload never saw this potentially dangerous request.
-1. Blocked requests (along with other gateway data) will be visible in the attached Log Analytics workspace. Execute the following query to show WAF logs, for example.
-
-   ```
-   AzureDiagnostics
-   | where ResourceProvider == "MICROSOFT.NETWORK" and Category == "ApplicationGatewayFirewallLog"
+   ```bash
+   # [This whole execution takes about 40 minutes.]
+   az network application-gateway stop -g rg-bu0001a0042-03 -n $APPGW_FQDN_BU0001A0042_03 && \ # first incident
+   az network application-gateway start -g rg-bu0001a0042-03 -n $APPGW_FQDN_BU0001A0042_03 && \
+   az network application-gateway stop -g rg-bu0001a0042-04 -n $APPGW_FQDN_BU0001A0042_04 && \ # second incident
+   az network application-gateway start -g rg-bu0001a0042-04 -n $APPGW_FQDN_BU0001A0042_04
    ```
 
-## Validate Cluster Azure Monitor Insights and Logs
+   :bulb: As an important note around scalability, please take into account that after a total failover is effective, your infrastructure in the hot backup region must be capable of handling 100% of the traffic. Therefore, the recommendation is to plan for this sort of events, and prepare all related backend systems to cope with it. It doesn't necessarily mean that you should purchase resources to keep them iddle waiting for these very rare outages. But you want to cover every piece of infrastructure to ensure they are elastic (scalable) enough to handle that sudden increase in load. For instance [Azure Application Gateway v2 autoscaling](https://docs.microsoft.com/en-us/azure/application-gateway/application-gateway-autoscaling-zone-redundant) will kick in under these circunstances along with the Cluster Autoscaler configured as part of the [AKS Baseline](https://github.com/mspnp/aks-secure-baseline). Something else you migth want to consider setting up is [Horizontal Pod Autoscaling](https://docs.microsoft.com/en-us/azure/aks/concepts-scale#horizontal-pod-autoscaler) and [Resource Quotas](https://docs.microsoft.com/en-us/azure/aks/operator-best-practices-scheduler#enforce-resource-quotas). The latter must be configured to request up to 2X the normal capacity if you plan for full region take over. Certainly to avoid cascade failures, this will require to also contemplate out-of-cluster resources that are affected by your workload (e.g. databases, external APIs, etc.). They also must be ready to absord as much load as you consider appropiate in your contingency plan.
```

Comment:
```suggestion
   :bulb: As an important note around scalability, please take into account that after a total failover is effective, your infrastructure in the remaining region must be capable of handling 100% of the traffic. Therefore, the recommendation is to plan for this sort of event, and prepare all related backend systems to cope with it. It doesn't necessarily mean that you should purchase resources to keep them idle waiting for these very rare outages. But you want to cover every piece of infrastructure to ensure they are elastic (scalable) enough to handle that sudden increase in load. For instance [Azure Application Gateway v2 autoscaling](https://docs.microsoft.com/azure/application-gateway/application-gateway-autoscaling-zone-redundant) will kick in under these circumstances along with the Cluster Autoscaler configured as part of the [AKS Baseline](https://github.com/mspnp/aks-secure-baseline). Something else you might want to consider setting up is [Horizontal Pod Autoscaling](https://docs.microsoft.com/azure/aks/concepts-scale#horizontal-pod-autoscaler) and [Resource Quotas](https://docs.microsoft.com/en-us/azure/aks/operator-best-practices-scheduler#enforce-resource-quotas). The latter must be configured to allow up to 2X the normal capacity if you plan for full region take over. Certainly to avoid cascade failures, this will require to also contemplate out-of-cluster resources that are affected by your workload (e.g. databases, external APIs, etc.). They also must be ready to absorb as much load as you consider appropriate in your regional outage contingency plan.
```